tools/cli/go-whisk-cli/commands/util.go

```diff
@@ -856,6 +856,10 @@ func getURLBase(host string, path string) (*url.URL, error)  {
         return nil, whiskErr
     }
 
+    if !strings.HasPrefix(host, "http") {
+        host = "http://" + host
```

Comment:
correct, updated the code. If we can't avoid deployment using https, we can just add one test. do you have any suggestion we can add more tests?